.github/workflows/NightlyTests.yml

```diff
@@ -310,6 +310,47 @@ jobs:
        run: |
          ${GITHUB_WORKSPACE}/debug-build/bin/python3 -m pytest tools/pythonpkg/tests/fast --skiplist TestMemoryLeaks
 
+  python-sqllogic:
+     name: Python SQLLogicTest Library (Linux)
+     runs-on: ubuntu-latest
+     needs: linux-memory-leaks
+     container: ubuntu:18.04
```

Comment:
Can we move this outside of an old Ubuntu VM and instead just run on `ubuntu-latest`?